requirements.txt

```diff
@@ -1 +1,2 @@
 mypy==0.971
```

Comment:
This file is only used to constant the requirements to specific versions. Please at to setup.cfg (under test)